apollo-federation/src/sources/connect/models/validation.rs

```diff
@@ -637,6 +703,7 @@ impl Location {
 }
 
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[non_exhaustive]
```

Comment:
```suggestion
```

I don't want the main consumer to ever forget to translate new codes into their string representations.

Reply:
Makes sense, I'll remove this.